client/Command/GetConfig/ConfigData.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) [2019] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact Novell, Inc.
+ *
+ * To contact Novell about this file by physical or electronic mail, you may
+ * find current contact information at www.novell.com.
+ */
+
+#ifndef SNAPPER_CLI_COMMAND_GET_CONFIGS_CONFIG_DATA_H
+#define SNAPPER_CLI_COMMAND_GET_CONFIGS_CONFIG_DATA_H
+
+#include <string>
+#include <vector>
+#include <map>
+
+#include "client/Command/GetConfig.h"
+#include "client/proxy.h"
+
+namespace snapper
+{
+    namespace cli
+    {
+
+	class Command::GetConfig::ConfigData
+	{
+
+	public:
+
+	    class Table;
+
+	    class Csv;
+
+	    class Json;
+
+	    ConfigData(const Command::GetConfig& command);
+
+	    virtual ~ConfigData();
+
+	    virtual std::string output() const = 0;
+
+	protected:
+
+	    enum Column { KEY, VALUE };
```

Comment:
Is this called "Attribute" for other commands?

Reply:
Yes. But for this command we are not selecting attributes exactly. For this reason I decided to name it "Columns" in this case. In fact, for json output it is ignored because json always need both, the key and its value.